zebra-chain/src/notes/sapling.rs

```diff
@@ -1,196 +1,44 @@
-//!
-#![allow(dead_code)]
+//! Sapling notes
 
-use std::{fmt, io};
+#![allow(clippy::unit_arg)]
+#![allow(dead_code)]
```

Comment:
should this allow be removed?

```suggestion
```

Reply:
`NotePlaintext` is not used yet, it will be an output type when notes are decoded from `EncryptedCiphertext` types, but that's not implemented yet.